RELEASE/scripts/autoscend/auto_island_war.ash

```diff
@@ -28,9 +28,8 @@ boolean warOutfit(boolean immediate)
 				return false;
 			}
 		}
-		return true;
 	}
-	else
+	if(get_property("auto_hippyInstead").to_boolean())

```

Comment:
Alright. I somehow got it into my head that java was one of those languages that applies an else to the last of nested ifs instead of obeying the {} symbols.
reverted this to an else as you said.